amivapi/utils.py

```diff
@@ -20,7 +21,7 @@
 from flask import g
 
 
-def token_urlsafe(nbytes=None):
+def token_urlsafe(nbytes=32):
```

Comment:
Note: This just brings our function up-to-date with the python implementation, which uses 32 bytes by default.